game_patch/multi/server.cpp

```diff
@@ -328,6 +339,9 @@ bool handle_server_chat_command(std::string_view server_command, rf::Player* sen
     else if (cmd_name == "stats") {
         send_private_message_with_stats(sender);
     }
+    else if (cmd_name == "hasmap" || cmd_name == "haslevel") {
```

Comment:
I wonder if we should merge words like this, instead of using `_`. For short commands it works but with 3 or more words it will be unreadable :thinking: WDYT? But we already have `nextmap` so I see why you made it like this.

Reply:
Indeed I made it this way to match `nextmap`.

I think using `_` would be OK (perhaps even preferred) if these commands were entered via console (because then tab complete would make it much easier). Entered as chat commands though, I think making them more simple to type very quickly is important, and `_` is a bit annoying for that purpose.

For short chat commands like this at least, I think omitting the `_` is best... and I also don't think making long chat commands would be good anyway, so the question of how to handle 3 words probably won't come up.